src/askui_runner/modules/core/infrastructure/files/s3.py

```diff
@@ -57,9 +57,11 @@ def upload_file(
                     **self.headers,
                     **headers,
                 },
+                stream=True,
                 timeout=REQUEST_TIMEOUT_IN_S,
             )
-            if response.status_code != 200:
+            # Workaround for upload limit of 10MB
+            if response.status_code not in [200, 413]:
```

Comment:
Seems like we don't raise an error anymore for too large content. Is it so that we still get a report even if there is a file that is too large (but simply with the file missing)?

In this case I think we should write a little more and extract this as nicely as possible with an indication when this can be removed...
```python
def isIgnoredError(response: Response) -> bool:
    """...<explanation of what function does including what is ignored and why>"""
    return response.status_code == 413
    

...
if response.status_code != 200:
   if not isIgnoredError(response):
      response.raise_for_status()
...
```